plasmapy/formulary/collisions.py

```diff
@@ -471,16 +625,21 @@ def impact_parameter(
         a macroscopic description is valid. This is used to recover the
         average ion density (given the average ionization and electron
         density) for calculating the ion sphere radius for non-classical
-        impact parameters.
+        impact parameters. ``z_mean`` is a required parameter if ``method`` is
+        ``"lsfullinterp"``, ``"hlsmaxinterp"``, or ``"hlsfullinterp"``.
 
     V : ~astropy.units.Quantity, optional
         The relative velocity between particles.  If not provided,
         thermal velocity is assumed: :math:`\mu V^2 \sim 2 k_B T`
         where `mu` is the reduced mass.
 
-    method: str, optional
-        Selects which theory to use when calculating the Coulomb
-        logarithm. Defaults to classical method.
+    method : str, optional
+        The method by which to compute the Coulomb logarithm.
+        The default method is the classical straight-line Landau-Spitzer method
+        (``"classical"`` or ``"LS"``). The other 6 supported methods are ``"ls_min_interp"``,
+        ``"ls_full_interp"``, ``"ls_clamp_mininterp"``, ``"hls_min_interp"``, ``"hls_max_interp"``, and
+        ``"hls_full_interp"``. Please refer to the docstring of `Coulomb_logarithm` for more information
+        about these methods.
```

Comment:
```suggestion
        (``"classical"`` or ``"LS"``). The other 6 supported methods are 
        ``"ls_min_interp"``, ``"ls_full_interp"``, ``"ls_clamp_mininterp"``, 
        ``"hls_min_interp"``, ``"hls_max_interp"``, and ``"hls_full_interp"``.
        Please refer to the docstring of `Coulomb_logarithm` for more
        information about these methods.
```
Another minor change for line length